lib/ui/BUILD.gn

```diff
@@ -18,14 +18,14 @@ source_set("ui") {
     "painting/canvas.h",
     "painting/gradient.cc",
     "painting/gradient.h",
+    "painting/image.cc",
+    "painting/image.h",
     "painting/image_decoding.cc",
     "painting/image_decoding.h",
     "painting/image_filter.cc",
     "painting/image_filter.h",
     "painting/image_shader.cc",
     "painting/image_shader.h",
-    "painting/image.cc",
```

Comment:
Are you using the GN format? We should really switch to that. Most of these files are likely violating the formatting guidelines.